### DISCUSSION TITLE
cleveldb tendermint

### DISCUSSION THREAD

**Discussion by sadioguindo** [tendermint/tendermint#8390] [Q&A]
salut,
j'ai un problème avec 
CGO_LDFLAGS="-lsnappy" make install TENDERMINT_BUILD_OPTIONS=cleveldb
make: aucune règle avec install

**Comment by creachadair** [comment]
Hi @sadioguindo. Based on that error message, there are a couple possible problems:

- Make sure your working directory contains the Tendermint `Makefile`. You generally have to build from the repository root.
- Check that you have a reasonably new version of the `make` tool.

If those don't fix it, please tell us more: What OS you're building on, what version of Tendermint, etc.

**Reply by sadioguindo** [reply]
J'ai essayé les deux solutions mais j'ai toujours la même erreur. 
Je travaille sur Linux version 20.04
J'ai pas choisi de version tendermint j'essaye de suivre les étapes du lien suivant: https://docs.tendermint.com/v0.33/introduction/install.html
Merci bien pour votre réponse c'est gentil.

**Comment by creachadair** [comment]
Hi, @sadioguindo. Tendermint v0.33 is quite old now. I recommend you try https://docs.tendermint.com/v0.35/introduction/install.html instead. The instructions are similar, but the newer version of the Makefile should work better. (Or at least I hope so)

**Reply by sadioguindo** [reply]
D'accord, je vais l'essayer en attendant vous n'avez pas d'autres recommandations à me faire pour pouvoir l'installer. Je dois faire mon soutenance dans peu de jours.
